clang/test/Driver/sycl-bfloat16-lib.cpp

```diff
@@ -0,0 +1,58 @@
+///
+/// Check if bfloat16 native and fallback libraries are added on Linux
+///
+
+// UNSUPPORTED: system-windows
+
+/// ###########################################################################
+/// test that no bfloat16 libraries are added in JIT mode
+// RUN: %clangxx -fsycl %s --sysroot=%S/Inputs/SYCL -### 2>&1 \
+// RUN:   | FileCheck %s -check-prefix=BFLOAT16
```

Comment:
Can you also add a test case that passes `-fsycl-targets=spir64`?  This is also a JIT compile, so no bfloat16 library should be linked.

Reply:
Yes, added test.